networkx/classes/digraph.py

```diff
@@ -970,7 +970,34 @@ def in_edges(self):
         in_edges : InEdgeView
             A view of edge attributes, usually it iterates over (u, v)
             or (u, v, d) tuples of edges, but can also be used for
-            attribute lookup as `edges[u, v]['foo']`.
+            attribute lookup as `edges[u, v]['foo']`. The *EdgeView family
+            of objects handle requests for all edges without any data. The
+            *EdgeDataView family of objects handle special requests: either
+            restricting to source nodes via nbunch or asking for edge data
+             via data. Also note the difference between G.edges and G.edges()
```

Comment:
Could you remove the space in front of `via data` and the lines below as this will mess up the rendering of docs.

The current docs rendered by the change are available at https://output.circle-artifacts.com/output/job/ab098af6-f76c-4bf9-a732-5c243aca96da/artifacts/0/doc/build/html/reference/classes/generated/networkx.DiGraph.in_edges.html#networkx.DiGraph.in_edges and as you can see it has created a new quote instead of keeping it in a paragraph.